src/style.css

```diff
@@ -1 +1,68 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+* {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
```

Comment:
This line sets the font family for all elements to Avenir and provides fallback fonts. However, it seems that the font 'Avenir' is not available on all systems and you have provided a custom font file for it. It would be better if you set the font-family where it's needed and provide a fallback there. Also, note that you need to provide a fallback font-family in case the main one doesn't work (from the checklist).